File: Sprint02-Backend/src/main/java/com/br/saks/Sprint02/sprint02/model/Risco.java
package com.br.saks.Sprint02.sprint02.model;

import java.util.List;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.OneToMany;
import lombok.Data;

@Data
@Entity
public class Risco {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false, length = 20)
    private String nome;

    @OneToMany
    @JoinColumn(name = "id_investimento")
    private List<Investimento> investimentos;
}

File: Sprint02-Frontend/lib/entities/risco.dart
import 'investimento.dart';



class Risco {
  int? id;
  String? nome;
  List<dynamic>? investimentos;

  Risco(
      {this.id, this.nome, this.investimentos});

  factory Risco.fromJson(Map<String, dynamic>json){
    return Risco(
        id: json['id'],
        nome: json['nome'],
        investimentos:
        json['investimentos'].map((investimento) => Investimento.fromJson(investimento)).toList());
  }
}